nav2_planner/src/planner_server.cpp

```diff
@@ -270,6 +275,13 @@ PlannerServer::computePlan()
     publishPlan(result->path);
 
     action_server_->succeeded_current(result);
+
+    auto elapsed_time_ms = (now() - start_time).to_chrono<std::chrono::milliseconds>().count();
```

Comment:
http://docs.ros2.org/dashing/api/rclcpp/classrclcpp_1_1Time.html duration has a type seconds that would be the same as this complex statement. `(now() - start_time).seconds()`